src/sentry/snuba/metrics/utils.py

```diff
@@ -155,6 +155,8 @@ def generate_operation_regex():
 
 OP_REGEX = generate_operation_regex()
 
+# Regex that is used to validate that an operation is a percentile operation in the form p**.
```

Comment:
it makes sence to document here valid values like:
```
p50_fcp
p50_lcp
p95
```

Reply:
From my understanding p50_* should not be supported, it is only an alias.

Reply:
but if RegExp is redundant so please remove a comment too.